src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/CurrentOrderHeaderDetail.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.probation.courtcaseservice.service.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@ApiModel("CurrentOrderHeaderDetail")
+@Getter
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CurrentOrderHeaderDetail {
+    @JsonIgnore
```

Comment:
Wht do we not return this field  ? Would prepare-a-case need it in future requests ?

Reply:
I wasn't sure whether to have the sentenceId field inside the currentOrderHeaderDetail or returned as a field as part of the SentenceResponse, so I ignored it from the CurrentOrderHeaderDetail and included the sentenceId as a field as part of the SentenceResponse instead, if you see what I mean.